x-pack/plugins/security_solution/public/common/components/guided_onboarding_tour/tour_config.ts

```diff
@@ -101,7 +106,8 @@ const alertsCasesConfig: StepConfig[] = [
     anchor: `[tour-step="${getTourAnchor(
       AlertsCasesTourSteps.reviewAlertDetailsFlyout,
       SecurityStepId.alertsCases
-    )}"] .euiTabs`,
+    )}"] span.euiTab__content`,
+    offset: 20,
```

Comment:
Fixes the issue with the `euiTabs` not expanding when tour step active @michaelolo24 
<img width="1440" alt="Screen Shot 2022-11-02 at 4 00 16 PM" src="https://user-images.githubusercontent.com/6935300/199611753-77141d6f-fd67-4098-8519-060dcefa1709.png">